tests/resources/telescope_positions-South-ground.ecsv

```diff
@@ -54,6 +54,10 @@
 #     REFERENCE: {VERSION: 1.0.0}
 # schema: astropy-2.0
 telescope_name position_x position_y position_z geo_code
+LSTS-01     -20.63     -64.84      34.00  A1A
+LSTS-02      80.04      -0.77      29.00  B0A
+LSTS-03     -19.39      65.22      31.00  A3A
+LSTS-04    -105.06       1.15      33.00  Y0B
```

Comment:
were telescopes missing here or did you expand to an upgraded array?

Reply:
Beta contains LSTs. We had Alpha before.